### PR TITLE
Added Sybase Support

### PR DESCRIPTION
**Pull request by sbitteker**
Tested and using with jconn2, jconn3, defauulting to jconn4 DS class.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject hikari-cp "1.7.5"
+(defproject hikari-cp "1.7.6"
```
Comment (tomekw) [inline project.clj]: Thank you @sbitteker but could you, please, revert this commit? I don't want to mix two things - adding a feature and bumping a version (and therefore preparing a release) are two separate things :)

File: README.md [hunk 2]
```diff
@@ -150,7 +150,36 @@ Custom translations of properties can be added by extending the
 
 (defn -main [& args]
   (jdbc/with-db-connection [conn {:datasource datasource}]
-    (let [rows (jdbc/query conn "SELECT * FROM table")]
+    (let [rows (jdbc/query conn "SELECT 0")]
```
Comment (tomekw) [inline README.md]: Why this change?

File: project.clj (hunk 1 above)
Comment (jstepien) [inline project.clj]: +1, or even rebase to remove this commit from the branch. That'd be the best solution. Thank you @sbitteker!

File: README.md (hunk 2 above)
Comment (sbitteker) [inline README.md]: select 0 is a simple working query sybase and postgres.  The equivalent in Oracle is select 0 from dual.

File: README.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ A Clojure wrapper to [HikariCP](https://github.com/brettwooldridge/HikariCP) - "
 Add the following dependency to your `project.clj` file:
 
 ```clojure
-[hikari-cp "1.7.5"]
+[hikari-cp "1.7.6"]
```
Comment (tomekw) [inline README.md]: @sbitteker please remove this :)

Comment (sbitteker) [inline README.md]: done

**Comment by jstepien** [comment]
@sbitteker Thanks for addressing the conflicts! @tomekw LGTY?

**Comment by tomekw** [comment]
@jstepien LGTY? :)

I'm happy, good to go.
